Sources/FluentUI_iOS/Components/Bottom Sheet/BottomSheetController.swift

```diff
@@ -667,13 +670,15 @@ public class BottomSheetController: UIViewController, Shadowable, TokenizedContr
         move(to: isExpanded ? .collapsed : .expanded, interaction: .resizingHandleTap)
     }
 
-    private func updateResizingHandleViewAccessibility() {
-        if currentExpansionState == .expanded {
+    private func updateResizingHandleViewAccessibility(for state: BottomSheetExpansionState) {
```

Comment:
This method was being called from the `didSet` method for `currentExpansionState` when the `currentExpansionState` is `.transitioning`. The accessibility properties would be set for the collapsed state via the `else` block and the sheet would read "Expand, Collapsed, Douple to tap expand" after the sheet was in the expanded state. The `currentExpansionState` is set again to the `.expanded` state but the updated accessibility properties are not set in time for VoiceOver to read them correctly.

 There is no real Accessibility/VoiceOver use case for the transitioning state. Let's only update the accessibility properties if the state is `expanded` or `collapsed` and no-op for other states.